README.md

```diff
@@ -1,37 +1,80 @@
-# ui
-This provides a graphical interface to run and build your GPTScripts.
+# Acorn Desktop
 
-![Chat Image](assets/chat.png)
+💫 **The best way to build, use, and share AI Assistants.** 💫 
 
-## Running
-The UI is baked directly into the [gptscript](https://docs.gptscript.ai/) CLI. To run the UI, simply run the following command:
+https://github.com/user-attachments/assets/b9cfa1c3-ba76-4193-84bf-115096b1a3ee
 
-```bash
-gptscript --ui
-```
+At the core of Acorn Desktop are **Assistant** and **Tools**. 
 
-You can also specify a specific script to run (locally or remote).
+**Tools** are Acorn Desktop's super power. They let the user and assistants interact with the rest of the world, including the user's workstation, APIs, and the Internet.
 
-```bash
-gptscript --ui github.com/gptscript-ai/llm-basics-demo
-```
+**Assistants** bring tools and the user together. They couple tools with context that affects its behavior, focus, and even personality.
+
+A tool is a self-contained piece of functionality that an assistant can use to accomplish the tasks that a user lays out for it. Tools themselves can be LLM instructions, code-based (in python, node.js, or bash), or integrations with external systems.
```

Comment:
Worth mentioning that a tool can be go as well?

Reply:
in the context of this app, no bc this is referring to the inline tools you can create and those cannot be go